pkg/api/job_runs.go

```diff
@@ -171,7 +204,7 @@ func JobRunRiskAnalysis(dbc *db.DB, jobRun *models.ProwJobRun) (apitype.ProwJobR
 // testResultsFunc is used for injecting db responses in unit tests.
 type testResultsFunc func(testName string, release, suite string, variants []string) (*apitype.Test, error)
 
-func runJobRunAnalysis(jobRun *models.ProwJobRun, compareRelease string,
+func runJobRunAnalysis(jobRun *models.ProwJobRun, compareRelease string, jobRunTestCount int, historicalRunTestCount int,
 	testResultsFunc testResultsFunc) (apitype.ProwJobRunRiskAnalysis, error) {
 
 	logger := log.WithFields(log.Fields{
```

Comment:
Continuing with my theme of nitpicking logging, could you do us a favor and move this contextual logger up in the the parent function, then pass it to runJobRunAnalysis as an argument. (maybe drop or adjust the "func" in each per your preference) Then you can use this for all the logging in the function above and we'll consistently know what job run the problems are coming from. It could then be passed to any other function we log from to make sure we know what we're looking at when we see those messages.

I think I should have done it this way originally.